Andreas dialog

grenasberg opened the pull request:
Bugfiksing i dialog_1 og dialog_3, All tekst i dialog_2 er nå oversatt!

grenasberg committed:
Update dialog_2.asm

oppdaterte noen linjer

grenasberg committed:
Update dialog_2.asm

grenasberg committed:
Update dialog_2.asm

mange oversettelser

grenasberg committed:
Update dialog_1.asm

for lang linje, skrev om dialogen

grenasberg committed:
Update dialog_3.asm

noen linjer var for lange

grenasberg committed:
Update dialog_2.asm

all tekst er oversatt

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1,10 +1,14 @@
 Dialog0A5::
-    db "You've got the  "
-    db "Bird Key!@"
+    db "Du fikk fugle-  "
```
```suggestion
    db "Du fikk Fugle-  "
```
Siden det heter Ansiktsnøkkelen med stor A?

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "Du fikk kom-    "
+    db "passet! Nå kan  "
+    db "du se hvor      "
+    db "kistenene og    "
```
En "ne" for mye, så jeg har omorganisert hele i forslagene under:
```suggestion
    db "kistene og Mare-"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "passet! Nå kan  "
+    db "du se hvor      "
+    db "kistenene og    "
+    db "Marerittet      "
```
```suggestion
    db "rittet skjuler  "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "du se hvor      "
+    db "kistenene og    "
+    db "Marerittet      "
+    db "skjuler seg!    "
```
```suggestion
    db "seg! Dette kom- "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "kistenene og    "
+    db "Marerittet      "
+    db "skjuler seg!    "
+    db "Dette kompasset "
```
```suggestion
    db "passet har en ny"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "Marerittet      "
+    db "skjuler seg!    "
+    db "Dette kompasset "
+    db "har en ny egen- "
```
```suggestion
    db "egenskap: En    "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "skjuler seg!    "
+    db "Dette kompasset "
+    db "har en ny egen- "
+    db "skap: En tone   "
```
```suggestion
    db "tone forteller  "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "Dette kompasset "
+    db "har en ny egen- "
+    db "skap: En tone   "
+    db "forteller deg om"
```
```suggestion
    db "deg om en nøkkel"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "har en ny egen- "
+    db "skap: En tone   "
+    db "forteller deg om"
+    db "en nøkkel er    "
```
```suggestion
    db "er gjemt i rom- "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "skap: En tone   "
+    db "forteller deg om"
+    db "en nøkkel er    "
+    db "gjemt i rommet  "
```
```suggestion
    db "met du går inn  "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -17,18 +21,31 @@ Dialog0A6::
+    db "forteller deg om"
+    db "en nøkkel er    "
+    db "gjemt i rommet  "
+	db "du går inn i!   "
```
Her glemte du også @
```suggestion
	db "i!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -142,9 +168,13 @@ Dialog0B3::
     db "HO HO HO!       "
-    db "I'm your bad guy"
-    db "this time!!     "
+    db "Jeg er slemming-"
```
Hva med «Jeg er dagens slemming»? Hehe

grenasberg replied:
monster of the week liksom?

grenasberg replied:
kan du foreslå
172 db "en på menyen i  "
173 db "dag!! HO HO HO!@"
?
jeg kan ikke foreslå på min egen PR tror jeg

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -341,74 +377,134 @@ Dialog0C1::
+    db "Ditt neste mål  "
+    db "er nordover, i  "
+    db "Symresumpen!!   "
+    db "Uhuu, ja!@"
```
```suggestion
    db "Uhuu, ja visst!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -779,54 +875,94 @@ Dialog0D0::
-    db "the Nightmares! "
-    db "Climb the stairs"
-    db "before you!@"
+    db "_…#####, du har "
```
Bør vel være mellomrom etter ellipse også
```suggestion
    db "_… #####, du har"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -779,54 +875,94 @@ Dialog0D0::
+    db "sies at du ikke "
+    db "kan forlate øya "
+    db "før du vekker   "
+    db "vindfisken_…    "
```
```suggestion
    db "Vindfisken_…    "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -836,37 +972,63 @@ Dialog0DA::
+    db "litt med oss og "
+    db "høre på når vi  "
+    db "jammer? For 300 "
+    db "rupier, kan du  "
```
```suggestion
    db "rupier kan du få"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -836,37 +972,63 @@ Dialog0DA::
+    db "høre på når vi  "
+    db "jammer? For 300 "
+    db "rupier, kan du  "
+    db "få sniklytte på "
```
```suggestion
    db "sniklytte på en "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -836,37 +972,63 @@ Dialog0DA::
+    db "jammer? For 300 "
+    db "rupier, kan du  "
+    db "få sniklytte på "
+    db "en uutgitt låt! "
```
```suggestion
    db "uutgitt låt!    "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1067,13 +1259,19 @@ Dialog0EE::
+    db "skjult snegle-  "
+    db "hus! Samler du  "
+    db "mange av disse, "
+    db "vil helt sikkert"
```
```suggestion
    db "vil det helt    "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1067,13 +1259,19 @@ Dialog0EE::
+    db "hus! Samler du  "
+    db "mange av disse, "
+    db "vil helt sikkert"
+    db "noe bra skje! @"
```
```suggestion
    db "sikkert skje noe"
    db "bra!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1023,36 +1194,57 @@ Dialog0E7::
+    db "Du fikk speil-  "
+    db "skjoldet! Nå kan"
+    db "du reflektere de"
+    db "strålene du før "
```
```suggestion
    db "strålene du ikke"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1023,36 +1194,57 @@ Dialog0E7::
+    db "skjoldet! Nå kan"
+    db "du reflektere de"
+    db "strålene du før "
+    db "ikke kunne      "
```
```suggestion
    db "kunne blokkere  "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1023,36 +1194,57 @@ Dialog0E7::
+    db "du reflektere de"
+    db "strålene du før "
+    db "ikke kunne      "
+    db "blokkere!@"
```
```suggestion
    db "før!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1095,16 +1293,22 @@ Dialog0F1::
-    db "Waterfall at the"
-    db "    Shrine@"
+    db "   Foss ved     "
+    db "   tempelet@"
```
```suggestion
    db "   Tempelet@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
-    db "chest?!  Where'd"
-    db "you learn ta do "
-    db "such a thing?!@"
+    db "Hei! Hva gjørr'u"
```
```suggestion
    db "Hei! Hva gjør'u "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
-    db "such a thing?!@"
+    db "Hei! Hva gjørr'u"
+    db "i kista mi?!    "
+    db "Hvor har'ru lært"
```
```suggestion
    db "Hvor har'u lært "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
+    db "Hei! Hva gjørr'u"
+    db "i kista mi?!    "
+    db "Hvor har'ru lært"
+    db "å gjøre sånt a?!@"
```
```suggestion
    db "å gjøre sånt'a?!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
 Dialog100::
-    db "You've got the  "
-    db "Full Moon Cello!@"
+    db "Du fikk full-   "
```
```suggestion
    db "Du fikk Full-   "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
-    db "You've got the  "
-    db "Conch Horn!@"
+    db "Du fikk         "
+    db "konkyliehornet!@"
```
```suggestion
    db "Konkyliehornet!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
 Dialog102::
-    db "You've got the  "
-    db "Sea Lily's Bell!@"
+    db "Du fikk sjø-    "
```
```suggestion
    db "Du fikk Sjø-    "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
 Dialog103::
-    db "You've got the  "
-    db "Surf Harp!@"
+    db "Du fikk bølge-  "
```
```suggestion
    db "Du fikk Bølge-  "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
 Dialog104::
-    db "You've got the  "
-    db "Wind Marimba!@"
+    db "Du fikk vind-   "
```
```suggestion
    db "Du fikk Vind-   "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
 Dialog105::
-    db "You've got the  "
-    db "Coral Triangle!@"
+    db "Du fikk korall- "
```
```suggestion
    db "Du fikk Korall- "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
-    db "You've got the  "
-    db "Organ of        "
-    db "  Evening Calm!@"
+    db "Du fikk kvelds- "
```
```suggestion
    db "Du fikk Kvelds- "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -1228,41 +1449,61 @@ Dialog0FE::
 Dialog107::
-    db "You've got the  "
-    db "Thunder Drum!@"
+    db "Du fikk torden- "
```
```suggestion
    db "Du fikk Torden- "
```

grenasberg committed:
Apply suggestions from code review

Gode innspill!

Co-authored-by: Tobias V. Langhoff <tobias@langhoff.no>

grenasberg committed:
Update dialog_1.asm

oversetter videre på dialog 1

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -97,38 +103,65 @@ Dialog005::
+    db "en sang_… Ja,   " 
+    db "sangen er       " 
+    db "«Balladen om    "
+    db "Vindfisken»@"
```
```suggestion
    db "Vindfisken»!@"
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -142,9 +168,13 @@ Dialog0B3::
-    db "I'm your bad guy"
-    db "this time!!     "
+    db "Jeg er slemming-"
+    db "en denne gangen!"
```
```suggestion
    db "en på menyen i  "
```

tobiasvl commented on revisions/N0/src/text/dialog_2.asm:
```diff
@@ -142,9 +168,13 @@ Dialog0B3::
-    db "this time!!     "
+    db "Jeg er slemming-"
+    db "en denne gangen!"
     db "HO HO HO!@"
```
```suggestion
    db "dag!! HO HO HO!@"
```

grenasberg committed:
Update dialog_1.asm

denne var ute av sync med norwegian, så jeg la inn manuelt det tobias hadde lagt inn etter at jeg branchet

grenasberg committed:
Apply suggestions from code review

Co-authored-by: Tobias V. Langhoff <tobias@langhoff.no>

grenasberg committed:
Update dialog_1.asm

mer dialog

grenasberg committed:
Update dialog_1.asm

enda mer

tobiasvl committed:
Merge branch 'norwegian' into andreas-dialog

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -197,11 +214,17 @@ Dialog00C::
+    db "Som vaskebjørn  "
+    db "er nesa mi      "
+    db "veeeeldig følsom"
+    db "for saker sånt  "
```
Dette er vel litt rar formulering? «Saker sånt som»

tobiasvl replied:
```suggestion
    db "for saker som   "
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -338,80 +361,133 @@ Dialog016::
+    db "Vil du gi meg 28"
+    db "rupier for min  "
+    db "hemmelige ting? "
+    db "   Gi  La vær<ask>"
```
```suggestion
    db "   Gi  La være<ask>"
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -338,80 +361,133 @@ Dialog016::
+    db "Hei der, tøffen!"
+    db "Jeg er Gale Tale"
+    db "og jeg selger en"
+    db "liten hemmelig  "
```
hemmelighet?

tobiasvl replied:
```suggestion
    db "liten hemmelig- "
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -338,80 +361,133 @@ Dialog016::
+    db "rupier for min  "
+    db "lille hemmelig- "
+    db "het_…           "
+    db "   Gi  La vær<ask>"
```
```suggestion
    db "   Gi  La være<ask>"
```

grenasberg committed:
Update dialog_1.asm

fra åsyn til trynet mitt

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -338,80 +361,133 @@ Dialog016::
+    db "idet du mister  "
+    db "alle hjertene!  "
+    db "Kom innom igjen "
+    db "annen gang,     "
```
```suggestion
    db "en annen gang,  "
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -426,28 +502,46 @@ Dialog023::
-    db "stress... Close "
-    db "your eyes and   "
-    db "relax...@"
+    db "La oss helbedre "
```
```suggestion
    db "La oss helbrede "
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -408,28 +502,46 @@ Dialog023::
-    db "of my face!@"
+    db "Hei din lus!    "
+    db "Tror du at du   "
+    db "slå meg?! Okei, "
```
```suggestion
    db "kan slå meg?!   "
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -408,28 +502,46 @@ Dialog023::
+    db "Hei din lus!    "
+    db "Tror du at du   "
+    db "slå meg?! Okei, "
+    db "gutter, få denne"
```
```suggestion
    db "Gutter, få denne"
```

grenasberg committed:
Apply suggestions from code review

Co-authored-by: Tobias V. Langhoff <tobias@langhoff.no>

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -179,11 +214,17 @@ Dialog00C::
+    db "er nesa mi      "
+    db "veeeeldig følsom"
+    db "for saker sånt  "
+    db "som støv og     "
```
```suggestion
    db "støv og pulver_…@"
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -320,80 +361,133 @@ Dialog016::
+    db "Jeg er Gale Tale"
+    db "og jeg selger en"
+    db "liten hemmelig  "
+    db "ting som vil gi "
```
```suggestion
    db "het som vil gi  "
```

grenasberg committed:
Apply suggestions from code review

Co-authored-by: Tobias V. Langhoff <tobias@langhoff.no>

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -179,11 +214,17 @@ Dialog00C::
+    db "veeeeldig følsom"
+    db "for saker som   "
+    db "støv og pulver_…@"
+    db "pulver_… @"
```
```suggestion
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -320,80 +361,133 @@ Dialog016::
-    db "    Give Don't<ask>"
+    db "Vil du gi meg 28"
+    db "rupier for min  "
+    db "hemmelige ting? "
```
```suggestion
    db "hemmelighet?    "
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -320,80 +361,133 @@ Dialog016::
+    db "Vil du gi meg 28"
+    db "rupier for min  "
+    db "hemmelige ting? "
+    db "   Gi  La være<ask>"
```
```suggestion
    db "    Gi   La være<ask>"
```

tobiasvl commented on revisions/N0/src/text/dialog_1.asm:
```diff
@@ -320,80 +361,133 @@ Dialog016::
+    db "rupier for min  "
+    db "lille hemmelig- "
+    db "het_…           "
+    db "   Gi  La være<ask>"
```
```suggestion
    db "    Gi   La være<ask>"
```

grenasberg committed:
Apply suggestions from code review

Co-authored-by: Tobias V. Langhoff <tobias@langhoff.no>
